index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="toolbar-container">
+      <div class="logo-container">
+        <div class="logo-item">
+          <a class="logo" href="#intro-section">My Logo</a>
+        </div>
+      </div>
+      <div class="nav-container">
+        <a href="#">
+          <img
+            class="nav-logo"
+            src="./images/header/Hamburger_icon.svg.png"
+            alt="logo" />
+        </a>
+      </div>
+    </header>
+    <main class="headline">
+      <section>
+        <div class="name-container">
+          <div class="name">
+            <h1>I’m Ali Hassan. Glad to see you!</h1>
```

Comment:
- [ ] Hello @mhassa204, please implement the requested change by the previous reviewer. You need to reduce the font size of your h1 tag so that your design closely matches the Figma 👍🏾